class/abstract-hd-segwit-p2sh-wallet.js

```diff
@@ -24,9 +24,10 @@ const { RNRandomBytes } = NativeModules;
 function ypubToXpub(ypub) {
   let data = b58.decode(ypub);
 
-  if (data.readUInt32BE() !== 0x049d7cb2) throw new Error('Not a valid ypub extended key!');
+  if (data.readUInt32BE() !== Number(`0x${mapperForEnvironment(config.networkName, 'ypub')}`))
+    throw new Error('Not a valid ypub extended key!');
   data = data.slice(4);
-  data = Buffer.concat([Buffer.from('0488b21e', 'hex'), data]);
+  data = Buffer.concat([Buffer.from(mapperForEnvironment(config.networkName, 'xpub'), 'hex'), data]);
```

Comment:
maybe instead of passing `config.networkName` to the function we could use it directly in the function. The function then could be named `getMasterPublicKeyPrefix('xpub');`

Reply:
done